certified-connectors/Impexium/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1850 @@
+              "type": "object",
+              "properties": {
+                "AdditionalFilter": {
+                  "type": "string",
+                  "description": "AdditionalFilter"
+                },
+                "Date": {
+                  "type": "string",
+                  "description": "Date"
+                }
+              },
+              "description": "Properties"
+            },
+            "Notifications": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "Action": {
+                    "type": "string",
+                    "description": "Action"
+                  },
+                  "OrderNumber": {
+                    "type": "string",
+                    "description": "OrderNumber"
+                  },
+                  "InvoiceNumber": {
+                    "type": "string",
+                    "description": "InvoiceNumber"
+                  },
+                  "ProductCode": {
+                    "type": "string",
+                    "description": "ProductCode",
+                    "title": ""
+                  },
+                  "ProductName": {
+                    "type": "string",
+                    "description": "ProductName"
+                  },
+                  "ProductType": {
+                    "type": "string",
+                    "description": "ProductType"
+                  },
+                  "Quantity": {
+                    "type": "string",
+                    "description": "Quantity"
+                  },
+                  "ItemPrice": {
+                    "type": "string",
+                    "description": "ItemPrice"
+                  },
+                  "CustomerType": {
+                    "type": "string",
+                    "description": "CustomerType"
+                  },
+                  "CustomerId": {
+                    "type": "string",
+                    "description": "CustomerId"
+                  },
+                  "FullName": {
+                    "type": "string",
+                    "description": "FullName"
+                  },
+                  "FirstName": {
+                    "type": "string",
+                    "description": "FirstName"
+                  },
+                  "LastName": {
+                    "type": "string",
+                    "description": "LastName"
+                  },
+                  "EmailAddress": {
+                    "type": "string",
+                    "description": "EmailAddress"
+                  },
+                  "RecordNumber": {
+                    "type": "string",
+                    "description": "RecordNumber"
+                  }
+                }
+              },
+              "description": "Notifications"
+            }
+          }
+        }
+      },
+      "post": {
+        "responses": {
+          "default": {
```

Comment:
@RjImpx Please 'replace' the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.